packages/e2e-tests/scripts/get-stats.ts

```diff
@@ -2,6 +2,17 @@ import { lstatSync, readFileSync, readdirSync } from "fs";
 import { join, relative } from "path";
 
 type Examples = typeof import("../examples.json");
+
+/**
+ * Note: typeof does not include optional props, so we need to
+ * cast to this instead.
+ */
+type AllExampleProps = {
+  recording: string;
+  buildId: string;
+  playwrightScript?: string;
+  requiresManualUpdate?: string;
```

Comment:
Why do you treat this as `string`? It's a `boolean` in the `examples.json` file.

Reply:
Mistake